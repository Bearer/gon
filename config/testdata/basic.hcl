source = ["./terraform"]
bundle_id = "com.mitchellh.test.terraform"

apple_connect {
  username = "mitchellh@example.com"
  password = "hello"
}

sign {
  application_identity = "foo"
}
