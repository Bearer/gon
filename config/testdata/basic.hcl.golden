(*config.Config)({
 Source: ([]string) (len=1 cap=1) {
  (string) (len=11) "./terraform"
 },
 BundleId: (string) (len=28) "com.mitchellh.test.terraform",
 Sign: (config.Sign) {
  ApplicationIdentity: (string) (len=3) "foo"
 },
 AppleConnect: (config.AppleConnect) {
  Username: (string) (len=21) "mitchellh@example.com",
  Password: (string) (len=5) "hello",
  Provider: (string) ""
 },
 Zip: (*config.Zip)(<nil>)
})
